File: Iniciante/1848.py
soma = 0;
cont = 0;

while (cont < 3):
  entrada = input();
  if (entrada == 'caw caw'):
    print(soma);
    cont += 1;
    soma = 0;
  else:
    olhos = list(entrada);
    for i in range(len(olhos)):
      if (olhos[i] == '*'):
        soma += 1*2**(2-i);

File: Iniciante/1848.js
let soma = 0;

for (let i = 0; i < lines.length; i++) {
  let entrada = lines[i];

  if (entrada == 'caw caw') {
    console.log(soma);
    soma = 0;
  } else {
    let olhos = entrada.split('');
    for (let j = 0; j < olhos.length; j++) {
      if (olhos[j] == '*') {
        soma += 1*2**(2-j);
      }
    }
  }
}
